locales/fr_FR/pcsx2_Main.po

```diff
@@ -38,7 +38,7 @@ msgstr ""
 
 #: common/src/Utilities/Exceptions.cpp:204
 msgid "Oh noes! Out of memory!"
-msgstr "Oups ! Mémoire surchargée !"
+msgstr "Oups ! Plus de mémoire !"
```

Comment:
Remove spaces before !'s

Reply:
French language has spaces before and after theses characters:  ? ! : ; % " + - / * =
See: https://www.blogdumoderateur.com/regles-ponctuation-typographie/